### ISSUE TITLE
Impossible to write float number into brush size text input (dot)

### ISSUE THREAD

**Issue by Q210108** [opentoonz/opentoonz#4337]
![image](https://user-images.githubusercontent.com/100626161/158458834-d2e119ce-6a60-47a1-ac05-0cea1a3a51f8.png)
As you can see on the picture, i cannot write dot into the text input

**Comment by RodneyBaker** [comment]
I see a comma there in your screenshot and not a period.  Did you mean to use a comma?

**Comment by artisteacher** [comment]
On my end, it only seems to be the original raster brush that can't use a decimal.

**Comment by Q210108** [comment]
> I see a comma there in your screenshot and not a period. Did you mean to use a comma?
> 
> Aside: I thought a change was made to allow for use of comma as well as period (for languages that use that instead) but perhaps my memory is faulty.
> 
> Related report: #3042

hi, the problem is that when i press dot, the imput does't write it, but if you see the number into the max input, there is a dot. the problem is technical.

**Comment by Q210108** [comment]
> On my end, it only seems to be the original raster brush that can't use a decimal.

no, if you see on the picture, the input max has dot, the only way to write dot is to move the cursor, but it s not accurate

**Comment by RodneyBaker** [comment]
We are likely a little confused by your screenshot because it shows the Accuracy option which is only an option for vector brushes.  What @artisteacher is stating is that when we are drawing on standard raster with the default brush it doesn't allow float values but only full integers (see attached image):

![image](https://user-images.githubusercontent.com/2576432/158558283-fdf74091-6d61-4992-b256-7cf6338d5e6c.png)

**Comment by artisteacher** [comment]
> What @artisteacher is stating is that when we are drawing on standard raster with the default brush it doesn't allow float values but only full integers

Exactly. On my end, the vector brush, standard smart raster brush, and my paint brushes all allow me to add decimal values. I can type them, they display as expected. I can't reproduce on macOS. Maybe this is OS specific?
